manifest/v1alpha/slo/validation.go

```diff
@@ -138,6 +235,49 @@ var specCompositeValidationRule = validation.NewSingleRule(func(s Spec) error {
 	return nil
 })
 
+// CompositeSpec objective may only occur once, and may not be mixed with CountMetricsSpec or RawMetricSpec
+var specCompositeObjectiveValidationRule = validation.NewSingleRule(func(s Spec) error {
+	if s.HasCompositeObjectives() && len(s.Objectives) > 1 {
```

Comment:
for slice length assertion you can use `validation.SliceLength` rule

Reply:
Rewrote to `SliceUnique()` with `When()`